src/css/popup.css

```diff
@@ -1,4 +1,8 @@
 /*To make the gap between the title and the statistic smaller*/
+.overlay:hover {
```

Comment:
Can you use id of the dom element instead of a class? Like us `#power-button:hover` or sth?